src/ts/modules/timer.ts

```diff
@@ -0,0 +1,57 @@
+export const timer = (id: string, deadline: string) => {
+    const addZero = (num:number) => {
+        if (num <= 9) {
+            return '0' + num;
+        } else {
+            return num;
+        }
+    };
+
+    const getTimeRemaining = (endTime:string) => {
+        const time = Date.parse(endTime) - Date.parse(new Date());
```

Comment:
var Date: DateConstructor
new () => Date (+4 overloads)
Argument of type 'Date' is not assignable to parameter of type 'string'.ts(2345)